storage/innobase/include/log0recv.h

```diff
@@ -742,9 +742,8 @@ void recv_reset_buffer();
 bool recv_sys_resize_buf();
 
 /** Parse log records from a buffer and optionally store them to a
-hash table to wait merging to file pages.
-@param[in]  checkpoint_lsn  the LSN of the latest checkpoint */
-void recv_parse_log_recs(lsn_t checkpoint_lsn);
+hash table to wait merging to file pages. */
+void recv_parse_log_recs();
```

Comment:
are we now accessing checkpoint_lsn from global recv_sys data structure?

Reply:
This comes from https://github.com/mysql/mysql-server/commit/85f9abf99075052964d6434737a7645c311b2990#diff-26f5d4b9822d8a85edd9ea715cde19e90347775be561b8919601cfcb3df8cb49 . We were referencing checkpoint_lsn as a parameter but not using it.

From what I can see, we only used it on 5.7 and just to print a debug message - https://github.com/mysql/mysql-server/blob/5.7/storage/innobase/log/log0recv.cc#L3247